pkg/policymutation/policymutation.go

```diff
@@ -462,6 +478,16 @@ func isKindOtherthanPod(kinds []string) bool {
 	return false
 }
 
+func hasAutogenKinds(kind []string) bool {
+	for _, v := range kind {
+		if v == "Pod" || strings.Contains(engine.PodControllers, v) {
```

Comment:
Why do we need to check podControllers?

Reply:
Because `autogen-controllers` should not be none if Pod Controllers are present.